### ISSUE TITLE
Konvertierung in PDF

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-org#711]
Wenn ich ein Dokument in PDF umwandle und gleichzeitig aber z.B. mit Libreoffice eine Calc-Tabelle aufhabe oder ein Dokument offen habe, muss ich das erst speichern, dann schließt es sich und erst dann ist ein Konvertierung möglich. Ist das so in Libreoffice (also kein BUG sondern ein feature??), Ist manchmal nervig, wenn man z.B. an einer Tabelle arbeitet und schnell in einer anderen Sache was konvertiert und rausschicken will.

**Comment by j-dimension** [comment]
Ja, unter Windows und macOS ist das so. Unter Linux nicht. Ich habe hier auch keine Möglichkeiten, "drumherum" zu arbeiten.

**Comment by stm9x9** [comment]
kann man nicht irgendwie aus der LO-portable Version eine extra LO-Instanz für das pdf-konvertieren aufmachen?

**Comment by j-dimension** [comment]
Das ist mal ne ziemlich gute Idee! Müsste man ausprobieren, ob das mit unoconv zusammen funktioniert.

**Comment by j-dimension** [comment]
Eigentlich gehört das Konvertieren auf den Server. Solange da noch Installationen der j-lawyer.BOX der ersten Generation genutzt werden, kann ich das aber nicht generell umsetzen. Dafür reicht deren RAM nicht aus.

**Comment by PBaumfalk** [comment]
Vielleicht passt das hierher:

Die erste Konvertierung des Tages in PDF generiert eine Fehlermeldung auf macOS. Die zweite Konvertierung funktioniert dann reibungslos. Ist auch schon was länger so:



Mit freundlichen Grüßen / Kindly Regards

Patrick Baumfalk
Rechtsanwalt / Lawyer


Mobil:    +49 1525 53 04 194
E-Mail:   patrick.baumfalk@kanzlei-baumfalk.de


Baumfalk - Rechtsanwaltskanzlei
Hauptstr. 195
50169 Kerpen

Tel.: +49 2273 40 68 504
Fax: +49 2273 59 03 565

E-Mail: info@kanzlei-baumfalk.de
Web: www.kanzlei-baumfalk.de

**Comment by PBaumfalk** [comment]
![Bildschirmfoto 2020-05-07 um 08 48 31](https://user-images.githubusercontent.com/50404378/81269213-4c93fa00-9049-11ea-8a64-fd1a5efebc53.png)

**Comment by stm9x9** [comment]
> Eigentlich gehört das Konvertieren auf den Server. Solange da noch Installationen der j-lawyer.BOX der ersten Generation genutzt werden, kann ich das aber nicht generell umsetzen. Dafür reicht deren RAM nicht aus.

Wieviel RAM braucht das denn?

**Comment by PBaumfalk** [comment]
Dann ist nach dem nächsten update die alte Kiste eben outdated.

Ich bin Apple User. So ist das hier immer.

Mit freundlichen Grüßen / Kindly Regards

Patrick Baumfalk
Rechtsanwalt / Lawyer


Mobil:    +49 1525 53 04 194
E-Mail:   patrick.baumfalk@kanzlei-baumfalk.de


Baumfalk - Rechtsanwaltskanzlei
Hauptstr. 195
50169 Kerpen

Tel.: +49 2273 40 68 504
Fax: +49 2273 59 03 565

E-Mail: info@kanzlei-baumfalk.de
Web: www.kanzlei-baumfalk.de

Sent by Mobile

**Comment by j-dimension** [comment]
> > Eigentlich gehört das Konvertieren auf den Server. Solange da noch Installationen der j-lawyer.BOX der ersten Generation genutzt werden, kann ich das aber nicht generell umsetzen. Dafür reicht deren RAM nicht aus.
> 
> Wieviel RAM braucht das denn?

LO nimmt schon gern mal zwischen 250 - 500 MB. Sind dann komplexe Dokumente mit vielen Grafiken zu konvertieren, wird schnell mehr.

**Comment by mmellenthin** [comment]
> Ja, unter Windows und macOS ist das so. Unter Linux nicht. Ich habe hier auch keine Möglichkeiten, "drumherum" zu arbeiten.

Wir haben das Problem ferner unter Ubuntu, also Linux.
Gibt es da mitlerweile einen Lösungsansatz?

**Comment by mmellenthin** [comment]
Exemplarisch übersende ich mal die Fotos des Problems.

Gruß Marcus Mellenthin

**Comment by j-dimension** [comment]
Mir ist diesbezüglich kein Problem mehr bekannt. Ich schließe.

**Comment by stm9x9** [comment]
mit 3.4 noch nicht probiert, aber bei 3.2 habe ich es nach wie vor, daß alle anderen Instanzen geschlossen werden müssen.
